.travis.yml

```diff
@@ -94,7 +94,6 @@ matrix:
       language: generic
       osx_image: xcode8.3
       before_script:
-        - pip install six
```

Comment:
I am not sure where we actually use this utility, so I removed it. It seems the OS X build and tests are passing.